src/utils/index.ts

```diff
@@ -285,3 +300,8 @@ export type GetDeepProp<T extends object, K extends string> = K extends keyof T
 export type ArrElement<ArrType> = ArrType extends readonly (infer ElementType)[]
   ? ElementType
   : never;
+
+
+  export const epochToDate = (epoch: number): Date => {
```

Comment:
👍 but I think we'd probably already have this somewhere else? 🤔

Reply:
on balancer tools. Will move all this utils to bleu ui